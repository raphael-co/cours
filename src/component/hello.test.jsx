import { render, screen } from "@testing-library/react"
import Hello from "./hello.jsx"

test("print hello world to the user", () => {
    render(<Hello/>);

    const hello = screen.getByText("Hello world");
    expect(hello).toBeInTheDocument();
    expect(hello).toMatchSnapshot();
})