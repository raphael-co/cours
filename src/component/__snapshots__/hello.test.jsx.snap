// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print hello world to the user 1`] = `
<p>
  Hello world
</p>
`;
